src/theory/strings/theory_strings_rewriter.cpp

```diff
@@ -1231,9 +1231,31 @@ Node TheoryStringsRewriter::rewriteMembership(TNode node) {
         }
       }
     }
+    else if (x.getKind() == STRING_CONCAT)
+    {
+      // (str.in.re (str.++ x1 ... xn) (str.* R)) -->
```

Comment:
@ajreynol, `str.* R` => `re.* R` or  did I miss something?

Reply:
You're right, that was a typo. Will fix.